opentelemetry-jaeger/src/exporter/mod.rs

```diff
@@ -110,6 +115,10 @@ impl trace::SpanExporter for Exporter {
 #[derive(Debug)]
 pub struct PipelineBuilder {
     agent_endpoint: Vec<net::SocketAddr>,
+    // There are many variations in which it's read unclear which is causing not to be.
```

Comment:
Can we add a to-do item here to address this when we clean up the features in this creates? I think maybe splitting the `wasm` collector into a different mod would help. Also, make those `*_collector_client` features additive. Meaning they will not override each other. And then we can probably have less complex feature combinations and a better understanding of why the linter complains

Reply:
Created #736